src/main.js

```diff
@@ -46,6 +46,8 @@ function printResults(results, format) {
   let formatter;
   if (format === 'json') {
     formatter = require('./formatters/json.js');
+  } else if (format == 'junit') {
+    formatter = require('./formatters/junit.js');
   } else if (!format || format == 'stylish') {
```

Comment:
```suggestion
  } else if (!format || format === 'stylish') {
```

Always use triple ===